### PR TITLE
 Explicitly mention available parsers upon Errors and on CLI #272

### PR DESCRIPTION
**Pull request by lewismc**
Hi @scopatz here is a patch to begin with. I am still not getting further but will try my best to keep them coming as I get further along.

### CODE REVIEW

File: xdress/astparsers.py [hunk 1]
```diff
@@ -357,7 +357,8 @@ def pick_parser(file_or_lang, parsers):
         ps = parsers[lang]
         ps = [p for p in ps if PARSERS_AVAILABLE[p.lower()]]
         if len(ps) == 0:
-            msg = "{0} parsers not available: {1}"
+            msg = "{0} parsers not available: {1} these can be specified on the command "
+            "line with the optional '-p' argument"
```
Comment (scopatz) [inline xdress/astparsers.py]: I think the string need to be in parentheses here....

**Comment by scopatz** [comment]
Thanks for opening this @lewismc

**Comment by lewismc** [comment]
ACK. Apologies about that

**Comment by scopatz** [comment]
Thanks!

**Comment by lewismc** [comment]
np

**Pull request merged**